nautobot/dcim/tests/test_views.py

```diff
@@ -2223,6 +2223,19 @@ def setUpTestData(cls):
             "controller_managed_device_group": ControllerManagedDeviceGroup.objects.first().pk,
         }
 
+    @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+    def test_vdc_panel_includes_add_vdc_btn(self):
+        """Assert Add Virtual device Contexts button is in Device detail view: Issue from #6348"""
+        device = Device.objects.first()
+        url = reverse("dcim:device", kwargs={"pk": device.pk})
+        response = self.client.get(url)
+        response_context = response.content.decode(response.charset)
```

Comment:
Will this be still needed if you move to usage of `assertInHtml` below? :thinking:

Reply:
Yes